examples/varianter_cit/params.ini

```diff
@@ -1,6 +1,18 @@
-[parameters]
-color: black, gold, red, green
-shape: square, triangle, circle
-state: liquid, solid, gas
-material: leather, plastic, aluminum
-coating: anodic, cathodic
+
+PARAMETERS
+color   [black,  gold, red, green ]
+
+shape[square, triangle, circle]
+state[liquid, solid, gas]
+material[leather, plastic, aluminum]
+coating[anodic, cathodic]
+
+p7[1,3,5,6,7]
```

Comment:
Again, because this is a sample parameter file, that people will look at and try to follow its logic, I'd go with more descriptive names only.  For instance:

```
PARAMETERS
color[black, gold, red, green ]
shape[square, triangle, circle]
state[liquid, solid, gas]
material[leather, plastic, aluminum]
coating[anodic, cathodic]

CONSTRAINTS
color != black || shape != square
color != black || shape != triangle
color != gold || coating != cathodic
material != aluminum || color != gold
```

Reply:
This is good idea, I will change that.